Web/Cookie.hs

```diff
@@ -104,6 +107,9 @@ renderCookie (k, v) = k `mappend` char8 '=' `mappend` v
 renderCookies :: Cookies -> Builder
 renderCookies = renderCookiesBuilder . map (byteString *** byteString)
 
+renderCookiesBS :: Cookies -> S.ByteString
```

Comment:
Please add a minor version bump to the cabal file, update the ChangeLog with the new version, and add a `@since` comment on the new identifiers. Thanks!

Reply:
added!